addons/missileguidance/stringtable.xml

```diff
@@ -193,5 +193,21 @@
             <Chinesesimp>循环切换开火模式</Chinesesimp>
             <Chinese>循環切換開火模式</Chinese>
         </Key>
+        <Key ID="STR_ACE_MissileGuidance_MCLOS_Up">
+            <English>MCLOS Steer up</English>
+            <German>MCLOS Nach oben steuern</German>
```

Comment:
`nach` is a preposition and is written lowercase

Reply:
Ist natürlich richtig, aber es sollte sich eher lesen wie "MCLOS: Nach oben steuern". MCLOS ist ja nicht das Objekt, welches du steuerst, sondern das System mit dem man steuert.

Reply:
Might aswell put the colon into it then, doesn't hurt and makes it clearer IMO.